docker-compose/monitor/otel-collector-config-connector.yml

```diff
@@ -29,7 +29,7 @@ service:
     traces:
       receivers: [otlp, jaeger]
       processors: [batch]
-      exporters: [spanmetrics, jaeger]
```

Comment:
[jaeger exporter is deprecated](https://github.com/open-telemetry/opentelemetry-specification/pull/2858), hence the switch to otlp exporter.